lib/routers/establishment/rops.js

```diff
@@ -41,13 +40,17 @@ function canUpdate(req, res, next) {
   next();
 }
 
-function setYear(req, res, next) {
-  set(req.body, 'data.year', 2021);
+function parseYear(req, res, next) {
+  let { year } = req.body.data;
+  year = parseInt(year || (new Date()).getFullYear(), 10);
```

Comment:
Is it a real use case that the year is undefined?

Reply:
The context for this being that it probably shouldn't be, and it would be better to explicitly require a year and throw some kind of error if one isn't provided.